### PR TITLE
🌱 Improve logging in machine manager

### PR DESCRIPTION
**Pull request by sailorvii**
The form should be key:value pair as the definition:
  func (l Logger) Info(msg string, keysAndValues ...interface{})

🐛 

**What this PR does / why we need it**:
A panic will occur.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

### CODE REVIEW

**Comment by furkatgofurov7** [comment]
/test-v1b1-ubuntu-integration
/retitle 🌱 Improve logging in machine manager

**Review by furkatgofurov7 — APPROVED** [review]
/approve
/cc @kashifest @smoshiur1237

**Comment by sailorvii** [comment]
> /approve /cc @kashifest @smoshiur1237

Thanks for your review.
Could you give another help that the integration test failed for:
+ make
bash: line 1: go: command not found
bash: line 1: go: command not found

I'm not clear on how to fix it.

**Comment by furkatgofurov7** [comment]
> Thanks for your review. Could you give another help that the integration test failed for:
> 
>     * make
>       bash: line 1: go: command not found
>       bash: line 1: go: command not found
> 
> 
> I'm not clear on how to fix it.

Hey, we have some issues with CI scripts atm, fix is on the way https://github.com/metal3-io/project-infra/pull/402, once merged, I will re-trigger integration tests here.

File: baremetal/metal3machine_manager.go [hunk 1]
```diff
@@ -530,9 +530,9 @@ func (m *MachineManager) Delete(ctx context.Context) error {
 		// Remove clusterLabel from BMC secret.
 		tmpBMCSecret, errBMC := m.getBMCSecret(ctx, host)
 		if errBMC != nil && apierrors.IsNotFound(errBMC) {
-			m.Log.Info("BMC credential not found for BareMetalhost", host.Name)
+			m.Log.Info(fmt.Sprintf("BMC credential not found for BareMetalhost %s", host.Name))
```
[the remark this answers is not in the document]
Comment (zhouhao3) [reply baremetal/metal3machine_manager.go:533]: Agreed, we should unify the format.

[the remark this answers is not in the document]
Comment (sailorvii) [reply baremetal/metal3machine_manager.go:533]: Sorry for missing line 519. I also agree to unify the format to "msg", key,value... I'll refresh the change.

[the remark this answers is not in the document]
Comment (mboukhalfa) [reply baremetal/metal3machine_manager.go]: Great work, Thanks @sailorvii

**Pull request merged**